src/components/headerImageCaption.tsx

```diff
@@ -75,18 +62,28 @@ const HeaderImageCaptionStyles = css`
 	}
 `;
 
+const svgStyle = css`
+	line-height: 55px;
+	font-size: 0;
+	svg {
+		width: ${basePx(2.7)};
+    	height: ${basePx(2.7)};
```

Comment:
`basePx` is deprecated, please use `1.35rem` here instead:

```suggestion
        width: 1.35rem;
        height: 1.35rem;
```